content/docs/api-reference/subscribers/meta.json

```diff
@@ -0,0 +1,14 @@
+{
+  "pages": [
+    "subscribers-controller_create-subscriber",
+    "subscribers-controller_patch-subscriber",
+    "subscribers-controller_remove-subscriber",
+    "subscribers-controller_get-subscriber",   
+    "subscribers-controller_search-subscribers",
+    "subscribers-controller_get-subscriber-preferences",
+    "subscribers-controller_update-subscriber-preferences",
+
+    "---Subscribers v1 ---",
+    "..."
```

Comment:
Let's maybe only include functionality that is not available in v2? For example credentials, and remove all the others like like create, path, get, etc...